pkg/stacks/executorinfo.go

```diff
@@ -54,23 +67,27 @@ func (eif *KubeExecutorInfoDiscoverer) Discover(ctx context.Context) (*ExecutorI
 	eif.mutex.Lock()
 	defer eif.mutex.Unlock()
 
-	if eif.Image != "" {
+	if eif.Image != "" && eif.ImagePullPolicy != "" {
```

Comment:
What's this do? What's the benefit of having both of the checks?

Reply:
ExecutorInfo is the image (and now imagepullpolicy) that the `stack unpack` job must run with. The Discoverer gathers this information from the running environment (presumably the `stack manage` process) through some self-identification requiring queries to the K8s API.

In debug settings, with no running pods, the Discoverer can be told (through environment variables) what the image (and now imagepullpolicy) should be.  Whether discovered from the running pod or provided through environment variables, these values become fields in the `stack unpack` `Job`.

The check at line 70 is a cache for the already discovered container information.

The idea with this change was to enforce that both Image and ImagePullPolicy are set.  I'll need to remove the ImagePullPolicy test since it makes sense to omit that for a default behavior, thanks for pointing this out through your comments.